systemtest/src/test/java/io/strimzi/systemtest/security/NetworkPoliciesST.java

```diff
@@ -281,11 +279,6 @@ void testNPWhenOperatorIsInDifferentNamespaceThanOperand(ExtensionContext extens
         Map<String, String> labels = new HashMap<>();
         labels.put("my-label", "my-value");
 
-        EnvVar operatorLabelsEnv = new EnvVarBuilder()
-            .withName("STRIMZI_OPERATOR_NAMESPACE_LABELS")
-            .withValue(labels.toString().replaceAll("\\{|}", ""))
-            .build();
-
```

Comment:
This should be used in the CO installation (IIRC), so it should be in the `SetupClusterOperator.SetupClusterOperatorBuilder()`, where you should add the env -> the `withExtraEnvVars`

Reply:
Got it.. trying